modules/cli/src/main/scala/scala/cli/commands/Package.scala

```diff
@@ -41,6 +41,7 @@ import scala.util.Properties
 object Package extends ScalaCommand[PackageOptions] {
   override def name                                                          = "package"
   override def group                                                         = "Main"
+  override def inSipScala                                                    = false
```

Comment:
Why remove this one from the SIP binary? Generating a JAR is a pretty basic task.